templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
I'm reading the code and I'm not seeing a way for this to work.  The cloud provider looks to create a LB with a name based on the UID of the service object that gets created.  This is very hard to predict.

Another option:
1) Create the ELB as part of cloud formation.  Don't set any instances behind it.
2) Pass in the id and CNAME of the ELB in to the instance
3) As part of the start up script, reconfigure the ELB to add the master instance.

This means we'd be calling the aws API from within the start up script.  That is a little wonky but at least then we have TLS end to end.